moped-editor/src/utils/activityLogFormatters/mopedPhasesActivity.js

```diff
@@ -61,6 +48,20 @@ export const formatPhasesActivity = (change, phaseList, subphaseList) => {
     }
   });
 
+  // if the changes array includes a deletion that supersedes any other changes
+  if (changes.includes("is deleted")) {
```

Comment:
@tillyw i'm not familiar enough with the activity log to know how these entries are built up, but I would be wary of keying on the `is deleted` string if it is a label could theoretically be changed at some point. if possible, it would nice to make this work by keying on the column name `is_deleted` instead.

Reply:
one idea would be a few lines above this, where we look for the differences, to to check if the field is is_deleted and stop looping. that way we arent tied to the label

Reply:
thanks for pointing that out! i updated the code to use the ``is_deleted`` key and pushed the delete return action prior to looping through any other fields. re-requesting review!